src/main/java/blackjack/domain/state/StateFactory.java

```diff
@@ -0,0 +1,20 @@
+package blackjack.domain.state;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.Cards;
+import blackjack.domain.state.finished.Blackjack;
+import blackjack.domain.state.running.Hit;
+import java.util.Arrays;
+
+public class StateFactory {
+
```

Comment:
이 클래스는 인스턴화를 막아주면 좋을 것 같아요! 
```suggestion
public class StateFactory {

    private StateFactory() { 
    }
```

`이펙티브 자바 item 4. 인스턴스화를 막으려거든 private 생성자를 사용하라.` 를 참고하면 좋을 것 같네요!